app/jobs/base.rb

```diff
@@ -102,30 +104,15 @@ def stop(exception:)
       end
 
       def self.raw_log(message)
-        @@logger ||=
-          begin
-            f = File.open "#{Rails.root}/log/sidekiq.log", "a"
-            f.sync = true
-            Logger.new f
-          end
-
-        @@log_queue ||= Queue.new
-
-        if !defined?(@@log_thread) || !@@log_thread.alive?
-          @@log_thread =
-            Thread.new do
-              loop do
-                @@logger << @@log_queue.pop
-              rescue Exception => e
-                Discourse.warn_exception(
-                  e,
-                  message: "Exception encountered while logging Sidekiq job",
-                )
-              end
-            end
```

Comment:
@davidtaylorhq This was introduced 6 years ago so I'm not sure if you remember why we did this but I don't think this is necessary when we can just write directly to the logger in the main thread.

Reply:
6 years... wow 👴 

The separate thread was implemented in response to [this perf concern](https://github.com/discourse/discourse/pull/7091#discussion_r261914510) during review of the PR back in 2019. But perhaps we were being a little over-cautious 🤷‍♂️ 

I'm happy to try using a normal `Logger` instead